walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * This file wraps some commonly used job related function as a reusable module.
+ */
+
+/**
+ * Gets the requirements list from the servlet
+ * (which gets it from the database).
+ * @return {Object} the requirements list in a key-value mapping format.
+ */
+function getRequirementsList() {
+    // TODO(issue/17): GET request to servlet to get from database
+    // returning some hardcoded values for now
+    return {
+      'O_LEVEL': 'O Level',
+      'LANGUAGE_ENGLISH': 'English',
+      'DRIVING_LICENSE_C': 'Category C Driving License',
+    };
+}
+
+/**
+ * Gets job detail from user input.
+ * @return {Object} containing the user inputs.
+ */
+function getJobDetailsFromUserInput() {
```

Comment:
do we need to pass `document` in as a param?

Reply:
according to experiment, no need